sqlx/sqlx_service.c

```diff
@@ -225,14 +225,18 @@ static void
 _patch_configuration_maxrss(void)
 {
 	const gint64 pre = sqliterepo_max_rss;
-	if (sqliterepo_max_rss <= 0) {
+	if (sqliterepo_max_rss == -1) {
```

Comment:
Dangerous! If someone puts -2, the code will consider the RAM is always exhausted.
```suggestion
	if (sqliterepo_max_rss < 0) {
```

Reply:
Good catch !